tools/jslint.js

```diff
@@ -116,6 +96,41 @@ if (cluster.isMaster) {
     return;
   totalPaths = paths.length;
 
+  // Check if outputting JSLint report to a file instead of stdout
+  i = process.argv.indexOf('-o');
+  if (i !== -1) {
+    if (!process.argv[i + 1])
+      throw new Error('Missing output filename');
+    var outPath = process.argv[i + 1];
+    var count = 0;
+    if (!path.isAbsolute(outPath))
+      outPath = path.join(cwd, outPath);
+    if (format !== 'tap') {
+      fd = fs.openSync(outPath, 'w');
+    }
+    outFn = function(str) {
+      if (format === 'tap') {
+        const outPathNumbered = path.join(path.dirname(outPath),
+          `${path.basename(outPath, '.tap')}-${count}.tap`);
+        fd = fs.openSync(outPathNumbered, 'w');
```

Comment:
Should that be `'a'`?  If I read the code right, the file gets reopened for every line of output.  Using `'w'` will truncate it every time.

Also, on the line above, I'd assign the filename to a variable for clarity, it's kind of hard to read right now.